test/parallel/test-cli-permission-deny-fs.js

```diff
@@ -126,3 +129,21 @@ const fs = require('fs');
   assert.strictEqual(status, 1);
   assert.ok(!fs.existsSync('permission-deny-example.md'));
 }
+
+{
+  const firstPath = path.sep + process.cwd().split(path.sep)[1];
+  const pwd = fixtures.path('permission', 'loader');
+  const file = path.join(pwd, 'index.js');
+  const { status, stderr } = spawnSync(
+    process.execPath,
+    [
+      '--experimental-permission',
+      `--allow-fs-read=${firstPath}`,
+      file,
+    ]
+  );
+  assert.ok(
+    stderr.toString().includes('resource: \'/etc/passwd\''),
```

Comment:
Can we use assert.match here? If the test fails, it would generate a much more useful error message.